### DISCUSSION TITLE
Feedback on `/docs/table/`

### DISCUSSION THREAD

**Discussion by Joshua-Renelli** [iTwin/iTwinUI#1039] [Feedback]
Hey guys,

Love the new changes to the documentation for the React components, it's nice to see all available props displayed and explained in a very modern looking UI.

I did however notice that there are no longer any examples of different component States. Just wondering if there are any plans to include those in the updated documentation as they were quite helpful.

Cheers,
Josh

**Comment by mayank99** [comment]
Yeah, almost all pages are currently missing some content. The plan is to go through all pages and add more useful examples and text. We are tracking the status in https://github.com/iTwin/iTwinUI/issues/961

**Reply by Joshua-Renelli** [reply]
Awesome, thank you for the clarification. 

Will keep an eye out for more content in the future! :)
